update-extensions.ts

```diff
@@ -126,18 +126,59 @@ function overrideDefaultValue (configuration: monaco.extra.IConfigurationNode) {
   }
 }
 
-// eslint-disable-next-line @typescript-eslint/no-explicit-any
-function applyI18n (object: any, i18n: Partial<Record<string, string | { message: string }>>): any {
-  if (object === null) {
-    return object
-  } else if (typeof object === 'string') {
-    return object.replace(/^%(.*)%$/g, (g, g1) => (i18n[g1] as { message?: string } | undefined)?.message ?? i18n[g1] ?? g1)
-  } else if (Array.isArray(object)) {
-    return object.map(item => applyI18n(item, i18n))
-  } else if (typeof object === 'object') {
-    return Object.fromEntries(Object.entries(object).map(([key, value]) => [key, applyI18n(value, i18n)]))
-  } else {
-    return object
+interface MessageBag {
+  [key: string]: string | { message: string, comment: string[] } | undefined
+}
+
+function isString (value: unknown): value is string {
+  return Object.prototype.toString.call(value) === '[object String]'
```

Comment:
please let me know if you remember why you had to do this instead of `typeof value === 'string'` :)

Reply:
no reason a257a082fe20e4c3485f81db1b6ad3189c529b37

(It doesn't change the outputed files)